administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -197,8 +197,24 @@ public function getUpdatedList($state = false, $all = false, $cleanup = false)
 			$client = ApplicationHelper::getClientInfo($pk->client_id);
 			$path = Path::clean($client->path . '/templates/' . $pk->template . base64_decode($pk->hash_id));
 
+			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
+
 			if (file_exists($path))
 			{
+				if ((int) $pk->created_date)
+				{
+					$created_date = new Date($pk->created_date);
+					$created_date->setTimezone($tz);
```

Comment:
These `setTimezone` calls should not be necessary.  Instantiating a new `Joomla\CMS\Date\Date` object internally converts the timestamp to UTC.